unitary/alpha/quantum_world.py

```diff
@@ -20,7 +20,8 @@
 from unitary.alpha.sparse_vector_simulator import PostSelectOperation, SparseSimulator
 from unitary.alpha.qudit_state_transform import qudit_to_qubit_unitary, num_bits
 import numpy as np
-import itertools
+from itertools import combinations
```

Comment:
Prefer `import itertools` then `itertools.combinations` below

"Use import statements for packages and modules only, not for individual types, classes, or functions."
https://google.github.io/styleguide/pyguide.html#22-imports